spec/lib/govuk_elements_form_builder/form_builder_spec.rb

```diff
@@ -48,6 +48,19 @@ def size(method, size)
       ]
     end
 
+    it 'supports attributes defined as a string' do
+      output = builder.send method, 'name', class: 'custom-class'
```

Comment:
The whole point of this PR is to **support** to pass the _attribute_ as a **string** (the issue is described [here](https://github.com/ministryofjustice/govuk_elements_form_builder/issues/68)), so that's exactly why I've defined this test.

The attribute `:name` (symbol) is varying in this test to `'name'` (string).